support/cas-server-support-redis-core/src/main/java/org/apereo/cas/redis/core/RedisObjectFactory.java

```diff
@@ -56,70 +61,72 @@
      * @return the redis connection factory
      */
     public static RedisConnectionFactory newRedisConnectionFactory(final BaseRedisProperties redis) {
-        val poolConfig = redis.getPool() != null
-            ? redisPoolConfig(redis)
-            : LettucePoolingClientConfiguration.defaultConfiguration();
+        LOGGER.debug("Loaded Redis configuration {}", redis);
```

Comment:
Set to TRACE, and only document/log things that are actually needed/useful and/or not obvious. Host/port usually are good candidates.

Reply:
ok so I should remove the toString and provide only usefull log manually ? Or I could keep and change the default log to TRACE ?

Reply:
Why going on TRACE whereas several modules are on debug ?

Reply:
> ok so I should remove the toString and provide only useful log manually ? Or I could keep and change the default log to TRACE ?

I suggest that you do both. Keep the log level at trace, and only log what might actually be useful. (For example, there is no reason to log pool configuration if pools are not defined).

>Why going on TRACE whereas several modules are on debug ?

Those are being selectively reviewed and switched over. I think trace makes more sense when you are tracing the execution of method/config, without offering additional explanations. Debug makes sense, when behavior changes and explanations are needed for decisions.

Reply:
> > ok so I should remove the toString and provide only useful log manually ? Or I could keep and change the default log to TRACE ?
> 
> I suggest that you do both. Keep the log level at trace, and only log what might actually be useful. (For example, there is no reason to log pool configuration if pools are not defined).
> 
If I understand well you're suggesting that I keep a toString but by defining attributes to show (without the password attribute) and that I use it for TRACE level, and that I add a debug on "basic" comportment changed by a custom configuration. Is it your view ? 

> > Why going on TRACE whereas several modules are on debug ?
> 
> Those are being selectively reviewed and switched over. I think trace makes more sense when you are tracing the execution of method/config, without offering additional explanations. Debug makes sense, when behavior changes and explanations are needed for decisions.

Ok got it !

Reply:
I've made a new purpose about logs and levels in a new commit.